zkevm-circuits/src/state_circuit/test.rs

```diff
@@ -104,7 +104,7 @@ fn test_state_circuit_ok(
 fn degree() {
     let mut meta = ConstraintSystem::<Fr>::default();
     StateCircuit::<Fr>::configure(&mut meta);
-    assert_eq!(meta.degree(), 9);
+    assert_eq!(meta.degree(), 12);
```

Comment:
If I'm not mistaken, going from 9 to 12 means we bump the extended k from 3 to 4, which increases the proving time.

This is an open discussion: are we OK with this degree?  If not, do we have ideas on how to bring it down in the future?

Reply:
You're right about `k`.

I think it should be possible to reduce the degree back down to 9, but I'd have to put some more work into (re)unifying the mpt lookups for Account and AccountStorage rows. I'd prefer to put this off for now :)

Reply:
> You're right about `k`.
> 
> I think it should be possible to reduce the degree back down to 9, but I'd have to put some more work into (re)unifying the mpt lookups for Account and AccountStorage rows. I'd prefer to put this off for now :)

Of course!  I've opened an issue to track this https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/772 :)